src/Day1.hs

```diff
@@ -84,11 +86,14 @@ updatePlayerPos Input{keys=keys} (x,y) = (x',y')
 tileMap :: State -> Map (Int, Int) Tile
 tileMap st = case gameMode st of
   GameMode'Start -> fromList []
-  GameMode'Play -> mergeTiles clear $ fromList [ ( (x,y), Tile Nothing Nothing (Just wh1) ) ]
+  GameMode'Play -> mergeTiles playerTile ( mergeTiles testDoors ( mergeTiles testWalls clear ) )
```

Comment:
The order of inputs for mergeTiles are backwards. Don't forget to update `grid-proto`.